content/projects/dags-with-airflow/_index.md

```diff
@@ -11,6 +11,10 @@ ready: true
 submission_type: repo
 tags:
 - apache-airflow
+- postgresql
+- sql
+- sqlite
```

Comment:
Airflow manages its own database. We dont touch it. So we can't add it as a tag, We aren't learning any sqlite from this project.

There is also no mentionof postgres in this project descriotion. Have you deided that the use learner should use postgres to store the PR data? Or is there an assumption here?